participant/prisma/seed.ts

```diff
@@ -27,15 +36,90 @@ export type SeedSubmissionFormsType = {
   [key: string]: SeedDataType | SeedDataType[];
 };
 
+export type SeedDocumentType = {
+  displayFilename: string;
+  filepath: string;
+  filesize: number;
+  filetype: string;
+};
+
 export type SeedAgencySubmissionsType = {
   submissionId: string;
   forms: SeedSubmissionFormsType;
+  documents: SeedDocumentType[];
 };
 
 export type SeedSubmissionsType = {
   [key: string]: SeedAgencySubmissionsType[];
 };
 
+// Get the presigned s3 url for an object in s3.
+// We cannot use getUrlFromS3() because of issues importing s3.server.ts
+// outside of remix because we are using file-type to get the filetype.
+// Instead, reproduce the function here.
```

Comment:
This package is no end of headaches

Reply:
I guess another option here (that you _should not feel compelled to take_) is to put the check file function in another module, making the way clear to import `getURLFromS3` from `s3.server.ts`

Reply:
Yah this package is 😞 . That's a good idea about separating out `getURLFromS3()`. I'll add an `@TODO`.